static/app/views/settings/project/projectOwnership/codeOwnerFileTable.tsx

```diff
@@ -62,7 +62,7 @@ export function CodeOwnerFileTable({
         `/projects/${organization.slug}/${project.slug}/codeowners/${codeowner.id}/`,
         {
           method: 'PUT',
-          data: {raw: codeownerFile.raw},
+          data: {raw: codeownerFile.raw, date_updated: new Date()},
```

Comment:
we're going to want to use an iso string here since passing a date to the json parser uses toString which looks like `Wed Oct 04 2023 11:56:45 GMT-0700 (Pacific Daylight Time)`
```suggestion
          data: {raw: codeownerFile.raw, date_updated: new Date().toISOString()},
```

the mock api isn't great for testing this